src/extension/content-script/onstart.ts

```diff
@@ -12,9 +12,6 @@ async function onstart() {
   }
 
   const account = await api.getAccount();
-  // window.webln
```

Comment:
Don't we need to inject this script for FF still?

Reply:
yup! need to decide fallback for firefox